ludwig/collect.py

```diff
@@ -321,11 +324,10 @@ def cli_collect_activations(sys_argv):
         help='disable TensorFlow from using multithreading for reproducibility'
     )
     parser.add_argument(
-        '-uh',
-        '--use_horovod',
-        action='store_true',
-        default=None,
-        help='uses horovod for distributed training'
+        "-b",
+        "--backend",
+        help="specifies backend to use for parallel / distributed execution",
+        choices=ALL_BACKENDS,
```

Comment:
I guess the default is None if we don't specify it, but maybe being explicit that the default is the local one may be easier for the users and for the help message?

Reply:
It's usually the local backend, but the tricky thing is that if the user runs with Horovod like: `horovodrun ludwig train ...` then the backend will be Horovod.  We can specify this in the help message if you think it would add clarity, but I think we should leave the value itself as `None` for this reason.

Reply:
Oh you are right, forgot about that. Yes specifying in the message that horovodrun makes it autodetect horovod, and that this overrides the autodetection is a good idea for clarity.